src/config.c

```diff
@@ -206,7 +206,8 @@ config_property_definition _interfaceDefinitions[] = {
 config_property_definition _soundDefinitions[] = {
 	{ offsetof(sound_configuration, title_music),						"title_music",					CONFIG_VALUE_TYPE_UINT8,		2,								NULL					},
 	{ offsetof(sound_configuration, sound),								"sound",						CONFIG_VALUE_TYPE_BOOLEAN,		true,							NULL					},
-	{ offsetof(sound_configuration, ride_music),						"ride_music",					CONFIG_VALUE_TYPE_BOOLEAN,		true,							NULL					},
+	{ offsetof(sound_configuration, ride_music),						"ride_music",					CONFIG_VALUE_TYPE_BOOLEAN,		false,							NULL					},
```

Comment:
I couldn't remember if it was in the original game or not so I just left it disabled by default. Want me to change it to true?